dev-client/src/screens/CreateProjectScreen/components/Form.tsx

```diff
@@ -163,12 +164,17 @@ export const EditForm = ({
   );
 };
 
-export default function Form({editForm = false, onInfoPress}: Props) {
+// TODO(performance): Adjust types
```

Comment:
What sort of adjustments are needed?

Reply:
Nothing major, just regular props type adjustments I omitted here to save a bit of time